cpp/src/slice2js/Gen.cpp

```diff
@@ -2558,13 +2558,17 @@ Slice::Gen::TypeScriptVisitor::visitExceptionStart(const ExceptionPtr& p)
         _out << epar << ";";
     }
 
-    for (const auto& dataMember : allDataMembers)
+    for (const auto& dataMember : dataMembers)
     {
+        _out << sp;
+        writeDocCommentFor(dataMember);
```

Comment:
We weren't emitting doc-comments for exception members.
I see no reason for this, since we're emitting them just fine for structs and classes.